cou/steps/execute.py

```diff
@@ -54,60 +55,73 @@ def normal(text: str) -> str:
         :return: text formatted.
         :rtype: str
         """
-        return Style.RESET_ALL + Fore.RED + text + Style.RESET_ALL
+        return Style.RESET_ALL + text + Style.RESET_ALL
 
-    return (
-        normal(parameter + " (")
-        + bold("c")
-        + normal(")ontinue/(")
-        + bold("a")
-        + normal(")bort/(")
-        + bold("s")
-        + normal(")kip:")
-    )
+    return normal(parameter + "Continue(") + bold("y") + normal("/") + bold("n") + normal("):")
 
 
 async def _run_step(step: BaseStep, interactive: bool) -> None:
-    """Run step and all its sub-steps.
+    """Run a step and all its sub-steps.
 
-    :param step: Plan to be executed on steps.
+    :param step: Step to be executed.
     :type step: BaseStep
     :param interactive: Whether to run upgrade step in interactive mode.
     :type interactive: bool
     """
-    logger.debug("running step %s", step)
-    await step.run()
+    if isinstance(step, UpgradeStep):
+        progress_indicator.start(step.description)
+        await step.run()
+        progress_indicator.succeed()
+    else:
+        await step.run()
 
     if step.parallel:
         logger.debug("running all sub-steps of %s step in parallel", step)
-        grouped_coroutines = (apply_plan(sub_step, interactive) for sub_step in step.sub_steps)
+        grouped_coroutines = (apply_step(sub_step, interactive) for sub_step in step.sub_steps)
         await asyncio.gather(*grouped_coroutines)
     else:
         logger.debug("running all sub-steps of %s step sequentially", step)
         for sub_step in step.sub_steps:
             logger.debug("running sub-step %s of %s step", sub_step, step)
-            await apply_plan(sub_step, interactive)
+            await apply_step(sub_step, interactive)
 
 
-async def apply_plan(plan: BaseStep, interactive: bool) -> None:
-    """Apply the plan for upgrade.
+async def apply_step(step: BaseStep, interactive: bool) -> None:
+    """Apply a step to execute.
 
-    :param plan: Plan to be executed on steps.
-    :type plan: BaseStep
+    :param step: Step to be executed.
+    :type step: BaseStep
     :param interactive:
     :type interactive: bool
     """
+    # adding a space at the end to better separate description with prompt options
+    # Example:
+    #   This is an upgrade step Continue(y/n)
+    description = step.description + " "
```

Comment:
It is not being used right now, but I think it is a good practice to have a correct fallback UI for steps with single-line description and needs to be prompted (very likely to be added in the future). Otherwise, when we implement that kind of step, the UI will need additional adjustment. 

I'm changing the comments to the following:
```
    # Default description style for prompt for steps with single line description
    # Example:
    #   This is an upgrade step Continue (y/n): 
    description_to_prompt = step.description + " "
```
Would this be enough for addressing your concern here?

Reply:
If there is no usage right now, I'm in favor of removing it and let the `\n` be the standard. When we have a situation where it prompts with single line, let's add it.

Reply:
After some thinking, I have moved the styling to `prompt` function and always display `Continue (y/n): ` on a new line.